tensilelite/Tensile/Contractions.py

```diff
@@ -502,6 +507,17 @@ def CompoundPredicates(cls, state, problemType):
         if ('WorkGroupMappingXCC' in state) and ('WorkGroupMappingXCCGroup' in state):
             rv += [cls("WorkgroupMappingXCCCheck", value=[state['WorkGroupMappingXCC'], state['WorkGroupMappingXCCGroup']])]
 
+        # TODO- To improve the perf of these non-multiples cases
+        if state['ProblemType']['SwizzleTensorA']:
+            rv += [cls('SwizzleTensorA', value=state['ProblemType']['SwizzleTensorA'])]
+            rv += [cls("Free0SizeMultiple", index=0, value=state['MacroTile0'])]
+            rv += [cls("BoundSizeMultiple", index=-1, value=state['DepthU'])]
+
```

Comment:
Free0SizeMultiple avoids Edge block.
BoundSizeMultiple avoids TailLoop.
Both pass validations but poor performance. We can use predicates to make sure they work on non-Edge and non-Tail for now.